src/Lesson7_1/Cat.java

```diff
@@ -0,0 +1,43 @@
+package Lesson7_1;
+
+public class Cat {
+    private String name;
+    private int appetite;
+    private boolean satiety; // сытость по ТЗ
+
+
+    public Cat(String name, int appetite) {
+        this.name = name;
+        this.appetite = appetite;
+        this.satiety = false;
+    }
+
+    public void eat(Plate plate) {
+        // кошки отказываются есть, если еды мало
+        if (satiety == true) {
```

Comment:
Синтаксис Java позволяет, если у Вас сравнивается булева переменная (значение в ней) со значением true, либо false, сразу прописать следующим образом и проверить логическое значение в переменной:
```suggestion
        if (satiety) {
```